data/posts/react-router-v7.md

```diff
@@ -0,0 +1,47 @@
+---
+title: React Router v7
+summary: React Router v7 brings all the great things you love from Remix back to React Router
+date: 2024-11-22
+image: /blog-images/headers/react-router-v7.jpg
```

Comment:
Doesn't super matter to me, but if you want to use this image for the OG image as well, just add this field

```suggestion
image: /blog-images/headers/react-router-v7.jpg
ogImage: /blog-images/headers/react-router-v7.jpg
```